src/components/Modals/BuySell/components/AssetSearch/AssetList.tsx

```diff
@@ -0,0 +1,34 @@
+import { ListProps } from '@chakra-ui/react'
+import { FixedSizeList } from 'react-window'
+import { Text } from 'components/Text'
+
+import { BuySellAction, CurrencyAsset } from '../../BuySell'
+import { AssetRow } from './AssetRow'
+
+type AssetListProps = {
+  handleClick: (asset: CurrencyAsset) => void
+  assets: CurrencyAsset[]
+  type: BuySellAction
+} & ListProps
+
+export const AssetList = ({ assets, type, handleClick }: AssetListProps) => {
+  return assets?.length === 0 ? (
```

Comment:
According to your typings, you don't need the `?` since the array can be empty, not undefined and `length` property exists on the array prototype i.e all arrays, even empty ones.
What about using `assets.length` and inverting the ternary?